metadata-db-app/src/migrations/055_setup_services_table.clj

```diff
@@ -0,0 +1,58 @@
+(ns migrations.055-setup-services-table
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private services-column-sql
+  "id NUMBER,
+  concept_id VARCHAR(255) NOT NULL,
+  native_id VARCHAR(1030) NOT NULL,
+  metadata BLOB NOT NULL,
+  format VARCHAR(255) NOT NULL,
+  revision_id INTEGER DEFAULT 1 NOT NULL,
+  revision_date TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL,
+  deleted INTEGER DEFAULT 0 NOT NULL,
+  user_id VARCHAR(30),
+  service_name VARCHAR(20),
```

Comment:
Seems too small - I know Simon mentioned changing variable name to 80 - I assume this will also change to 80.